### PR TITLE
Allow to set ssh tunnel optional

### PR DESCRIPTION
**Pull request by Dudesons**
Hi @r4fek,

I can't try with ssh tunnel in my setup if you can say me if I don't break this feature

### CODE REVIEW

**Comment by r4fek** [comment]
Hi @Dudesons!

Thanks for this MR! I'll check tomorrow morning and merge it if it's OK. 
Could you tell me more about your setup and why do you use socks proxy?

**Comment by Dudesons** [comment]
Oki.
Our scylla cluster is deployed on aws on private subnets.
This cluster is accessible only if the instance have the good security group.
We have a bastion host to manage our instances with ansible or other tools.

So we start a ssh proxy socks and we set http_proxy and https_proxy to the proxy socks and run scli.

**Comment by Dudesons** [comment]
HI @r4fek ,

Do you have an ETA for the review ?

**Comment by r4fek** [comment]
Hi @Dudesons.

Sorry for the delay, I'm on unexpected vacation right now.

File: scli/main.py [hunk 1]
```diff
@@ -84,8 +85,27 @@ def destroy_ssh_tunnels():
...
+@click.option('-p', '--ssh_pass', is_flag=True,
+              help='Use this flag if your SSH key is protected by password')
+@click.option('-l', '--log_to', help='Where to store logs from the client')
+@click.pass_context
+def repair(ctx, host, keyspace, table, hosts, exclude, dc, local, ssh_username, ssh_pkey, ssh_pass, log_to):
+    if host is None:
+        click.echo('Either --host or SCYLLA_HOST env should be provided')
+        raise click.Abort()
+
+    if ssh_username != "":
```
Comment (r4fek) [inline scli/main.py]: I don't like this part (initializing ssh tunnel) being repeated this way for every command.

Comment (Dudesons) [reply scli/main.py]: Ok @r4fek  say me if you prefer with the commit `6d23a5a`

**Review by r4fek — APPROVED** [review]

File: scli/main.py (hunk 1 above)
Comment (r4fek) [reply scli/main.py]: Sorry again for the delay. Looks good!

**Pull request merged**